frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx

```diff
@@ -79,7 +79,7 @@ function CellInfoSideBar({
       if (Date.now() - hoverStartTime > 2 * 1000) {
         track(event, payload);
       }
-    }, [hoverStartTime]);
+    }, [event, payload]);
```

Comment:
why did we remove `hoverStartTime`?

Reply:
ESLint says `hoverStartTime` is not needed to be in the dep list, since it's a variable from an outer scope instead of a variable from the functional scope `useHandleHoverEnd` like `event` and `payload`. 

Essentially, at the time of the invocation of the function like `handleFmgHoverEnd`, it will use `this` from component scope and access the latest value of `hoverStartTime` as intended

<img width="738" alt="Screenshot 2023-08-21 at 2 20 18 PM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/63e1b6db-6686-44d6-bf82-48140cd1d9e3">

If we want to add `hoverStartTime` to the dep list, `hoverStartTime` needs to be passed in as an argument like `event` and `payload`, since primitives are passed in as values in the functional scope, which will be stale if it's not included in the dep list